book/src/introduction/quickstart.md

```diff
@@ -0,0 +1,46 @@
+# Quickstart
+
+Trin runs on Linux, MacOS, and Windows. There are two ways to run it: download a binary executable, or install it from source.
+
+Let's download the binary executable.
+
+## Download an executable
+
+The github repository hosts the binaries. Download the latest release for your platform from the [releases page](https://github.com/ethereum/trin/releases).
+
+Extract the compressed file to get the `trin` executable.
+
+### Extraction on Linux / MacOS
+
+The binary is compressed in a tarball, so first we need to extract it.
+
+For example, to extract version 0.1.0-alpha.51:
+
+```sh
+tar -xzvf trin-v0.1.0-alpha.51-x86_64-unknown-linux-gnu.tar.gz
+```
+
+You now have a `trin` executable in the current directory.
+
+## Run trin
+
+Launch the executable with 2GB local disk space:
+```sh
+./trin --portal-subnetworks state,history --mb 2000
```

Comment:
Maybe I'll delay merging this PR, to enable state by default and then remove this flag